### ISSUE TITLE
Missing PCK file error

### ISSUE THREAD

**Issue by YeldhamDev** [flathub/com.orama_interactive.Pixelorama#2]
Can't open the program, it shows the error and closes:
`Error: Could not load game data at path '.'. Is the .pck file missing?`

**Comment by Calinou** [comment]
This is really strange, as the runner script explicitly specifies the path of the main pack to use: https://github.com/flathub/com.orama_interactive.Pixelorama/blob/2dc044e97047af302340e1946d12f30694f6bf36/com.orama_interactive.Pixelorama.yaml#L33

**Comment by OverloadedOrama** [comment]
There was an issue when I updated the flatpak to 0.6.2 where the .pck file could not be located, but that was because the content was located in a directory inside the .zip file. That build failed and was not published, and I fixed it immediately by updating the .zip file, so this shouldn't be the issue, I think.

Could it be that Godot 3.2 changed something? Or because in 0.6.2 we use `OS.get_executable_path().get_base_dir()` instead of "." for the root directory (to load Brushes & Palettes)? Even though I don't think it should affect finding the .pck file.
